Tests/WolvenKit.Utility/DumpInfo.cs

```diff
@@ -336,23 +337,34 @@ private void MergeStrings(bool factOnly)
                         ext = hashService.GetGuessedExtension(dc.Hash);
                     }
 
-                    if (ext == ".questphase")
+                    if (ext == ".questphase" || ext == ".scene")
```

Comment:
We could use the enum.tostring here (actually, we should make a pass and refactor all hardcoded extensions to enums)